### ISSUE TITLE
BUG: cant install helm on Kubernertes 1.26

### ISSUE THREAD

**Issue by Glavnyuk** [alephdata/aleph#2998]
new version of  Kubernetes 1.26  dont support   autoscaling/v2beta1, need use  autoscaling/v2 in helm charts templates

**Comment by stchris** [comment]
First step tested in #3045, results in:

```
Error: UPGRADE FAILED: error validating "": error validating data: [ValidationError(HorizontalPodAutoscaler.spec.metrics[0].resource): unknown field "targetAverageUtilization" in io.k8s.api.autoscaling.v2.ResourceMetricSource, ValidationError(HorizontalPodAutoscaler.spec.metrics[0].resource): missing required field "target" in io.k8s.api.autoscaling.v2.ResourceMetricSource]
```

**Comment by stchris** [comment]
@Glavnyuk sorry you had to wait this long, but we finally released a fix for this with 3.15.4: https://github.com/alephdata/aleph/releases/tag/3.15.4
